manifests/init.pp

```diff
@@ -1,7 +1,13 @@
-class networkd {
-  service { "systemd-networkd":
-    ensure  => "running",
-    enable  => "true",
+#
+# Class networkd
+#
+class networkd (
+  Boolean $purge = true,
+
```

Comment:
stray newline. if we're going to fix style issues, we should clearly also avoid this empty line :-)

Reply:
Fair enough, got rid of the newline and another 2 newlines at the end of the network define.